paasta_tools/cli/utils.py

```diff
@@ -305,7 +305,7 @@ def guess_instance(service, cluster, args):
     else:
         try:
             instances = list_all_instances_for_service(
-                service=service, cluster=cluster, instance_type=None, soa_dir=args.yelpsoa_config_root)
```

Comment:
I'm missing it. The reason `local-run` does need it to help enhance the guess about which instance to use if you have specified the cluster.